### PR TITLE
bike profile should allow reverse oneway roads and set walking speed

### LINKED ISSUE #196: Allow oneways and foot-only roads for bike
http://graphhopper.com/maps/?point=49.952383%2C11.578732&point=49.953391%2C11.578404&vehicle=bike

Allow opposite oneways and foot-only roads and set `get_off_bike` to `true`. But assign not only low speed like 5km/h but also low priority.

### PR DESCRIPTION
**Pull request by karussell**
This PR fixes #196 with the help of #2760 and adapts get_off_bike accordingly.

To enable this feature you have to use custom model and adapt it or use the provided bike.json for it.

### CODE REVIEW

**Comment by jp-lopez** [comment]
Thanks :100:

**Review by jp-lopez — APPROVED** [review]
Good Job. I test it locally on my machine and it works. 

```
  graph.vehicles: roads,bike
  graph.encoded_values: max_width,max_height,average_slope,max_slope
  
    profiles:
    - name: bike
      vehicle: roads
      weighting: custom
      custom_model_file: bike.json
  
```
image
image

File: core/src/test/java/com/graphhopper/GraphHopperTest.java [hunk 1]
```diff
@@ -2504,16 +2500,6 @@ public void testBarriers() {
...
-        {
-            // the bollard blocks the road for bikes, and we need to take a big detour. note that this bollard connects
-            // two ways
-            GHResponse bikeRsp = hopper.route(new GHRequest(51.257709, 12.309269, 51.257594, 12.308882).setProfile("bike"));
-            assertEquals(1185, bikeRsp.getBest().getDistance(), 1);
-            // pedestrians can just pass the bollard
-            GHResponse footRsp = hopper.route(new GHRequest(51.257709, 12.309269, 51.257594, 12.308882).setProfile("foot"));
-            assertEquals(28, footRsp.getBest().getDistance(), 1);
-        }
-
```
Comment (easbar) [inline core/src/test/java/com/graphhopper/GraphHopperTest.java:2516]: I think this test was testing some general barrier-node logic, not really related to any specific vehicles. Did it fail for this PR, and do you know why? I could try to find another example for the same case if this is not a barrier anymore or something.

**Comment by ratrun** [comment]
In my test this new functionality does not work. Opposite oneways are still avoided. I noted that in the "Show routing graph" view of edges the new `backward_bike_access` encoded value does not show up. In contrast to that problem the `bike_access` value is present.
I checked the configuration against the examples above and do not find anything I'm missing. What might I be doing wrong?

**Comment by ratrun** [comment]
I'm using the latest master branch on my local setup. I did change the config file. This is the setup: `graph.vehicles: roads,bike` and 

```
    - name: bike
      vehicle: bike
      weighting: custom
      custom_model_file: bike.json
```

**Pull request merged**